src/celeritas/inp/SurfacePhysics.hh

```diff
@@ -0,0 +1,206 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/inp/SurfacePhysics.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <algorithm>
+#include <map>
+
+#include "corecel/Types.hh"
+#include "corecel/cont/Range.hh"
+#include "corecel/math/SoftEqual.hh"
+#include "geocel/Types.hh"
+#include "celeritas/inp/Grid.hh"
+
+namespace celeritas
+{
+namespace inp
+{
+//---------------------------------------------------------------------------//
+// SURFACE DESCRIPTION: Reflectivity and models for surface normals.
+//---------------------------------------------------------------------------//
+/*!
+ * Surface reflectivity index, which can be a user-defined Grid, which is a
+ * function of energy (wavelength), or constant.
```

Comment:
I think these should be `XReflectivity` since we have `XRoughness`. ReflectionGrid sounds like it's a `Grid ` about reflection.